ranger/core/main.py

```diff
@@ -471,7 +508,7 @@ def allow_access_to_confdir(confdir, allow):
         else:
             LOG.debug("Created config directory '%s'", confdir)
         if confdir not in sys.path:
-            sys.path[0:0] = [confdir]
+            sys.path.insert(0, confdir)
```

Comment:
I like this. Way less fancy-shmancy, way more clear re intent.

Reply:
Yah I didn't know what that line did at first XD although it is a neat bit of syntax